ocaml/xapi/pvs_proxy_control.ml

```diff
@@ -129,7 +140,14 @@ let start_proxy ~__context vif proxy =
 	with e ->
 		let reason =
 			match e with
-			| Xapi_pvs_cache.No_cache_sr_available -> "no PVS cache SR available"
+			| Xapi_pvs_cache.No_cache_sr_available ->
+				let body = Printf.sprintf "No cache storage found for site:%s on the host:%s"
```

Comment:
Let's make this "Unable to setup PVS-proxy %s for VIF %s: no cache storage found on PVS-site %s for host %s."